force-app/main/default/lwc/serviceScheduleCreator/__tests__/serviceScheduleCreator.test.js

```diff
@@ -0,0 +1,26 @@
+import { createElement } from "lwc";
```

Comment:
Really like the fact that you take the time to write jest tests for the components that you create. Need to make that as part of my checklist when I am writing new components..